love-letter-app/src/app/globals.css

```diff
@@ -126,9 +107,97 @@ body {
 
 #myCard {
   transition: transform 0.3s ease;
-  
 }
 
 .flipped {
   transform: rotateY(180deg);
 }
+
+.creation-component {
+  display: flex;
+  flex-direction: column;
+  background-color: #fafafa;
+  width: 80%;
+  max-width: 800px;
+  margin: 20px;
+  height: 40%;
+  max-height: 323;
+  border-radius: 10px;
+  border: 1px solid black;
+}
+
+.creation-component-header {
+  display: flex;
+  flex-direction: row-reverse;
+  align-items: center;
+  padding-right: 10px;
+  height: 40px;
+  border-bottom: 1px solid black;
+  font-family: work-sans, sans-serif;
+  font-size: 18px;
+  font-weight: 500;
+}
+
+.creation-component-body {
+  /* position: relative; */
+  overflow: hidden;
+  height: 100%;
+  position: relative;
+}
+
+.svg-container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-items: center;
+  overflow-y: auto;
+  height: 100%;
+  width: 100%;
+  padding: 10px;
+  border-radius: 10px;
+}
+
+.message-container {
+  height: 100%;
+  width: 100%;
+  padding: 10px;
+  font-family: work-sans, sans-serif;
+  font-size: 16px;
+  font-weight: 400;
+  border-radius: 10px;
+  background-color: #fafafa;
+  /* remove resizing */
+  resize: none;
+}
+
+.message-container:focus {
+  outline: none;
+}
+
+.create-button {
+  position: absolute;
+  bottom: 0;
+  right: 0;
+  margin: 15px;
+  /* then lets move the button to be inside the parent */
+  border-radius: 10px;
+  border: 1px solid black;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 102px;
+  height: 49px;
+  cursor: pointer;
+  font-family: work-sans, sans-serif;
```

Comment:
also variable for font-family. or perhaps you could define font-family somewhere at a top component or root (do some research), and then every child component can inherit same font-family